content/docs/apps/tic.md

```diff
@@ -0,0 +1,14 @@
+---
+menu:
+  docs:
+    parent: advanced
+title: Restricting access
```

Comment:
Could this more a slightly more specific title? "Restrict access by IP for TIC" ?

Reply:
I am thinking we want to split this into two docs... One will be about TIC compliance for cloud.gov in general, which then refers to this page for directions about configuring the "restrict client request origin for your domain" feature. That scopes this doc more tightly to what it is, while enabling us to give lots of hits for "TIC" in searches in the other doc.

Reply:
Maybe "Restrict administrative access by IP" ?